web-registry/src/hooks/useMsgClient.ts

```diff
@@ -4,8 +4,17 @@ import { StdFee, DeliverTxResponse } from '@cosmjs/stargate';
 import { useLedger } from '../ledger';
 import { Sender } from '../lib/wallet';
 
+interface MessageData {
```

Comment:
I think that'd be a better naming since fee and memo aren't really related to messages
```suggestion
interface TxData {
```

Reply:
done